executor/adapter.go

```diff
@@ -464,10 +464,10 @@ func (a *ExecStmt) handleNoDelay(ctx context.Context, e Executor, isPessimistic
 		// `rs.Close` in `handleStmt`
 		if sc != nil && rs == nil {
 			if sc.MemTracker != nil {
-				sc.MemTracker.DetachFromGlobalTracker()
+				sc.MemTracker.Detach()
```

Comment:
Why change `DetachFromGlobalTracker` into `Detach` here?

Reply:
the current tracker chain is analyze-> stmt -> AnalyzeGlobal -> Global. parent is not global anymore.